extensions/amp-consent/0.1/consent-ui.js

```diff
@@ -362,14 +372,20 @@ export class ConsentUI {
         data['initialHeight'].indexOf('vh') >= 0
       ) {
         const dataHeight = parseInt(data['initialHeight'], 10);
+        // Set initialHeight to max height fallback if applicable
+        this.initialHeight_ =
+          dataHeight >= 80 ? MAX_INITIAL_HEIGHT : this.initialHeight_;
```

Comment:
Why `80`? Name this number as a constant if arbitrary, for future context.

Reply:
80 is the `MAX_INITIAL_HEIGHT`

Reply:
Meaning, `maxInitialHeightForModal` or how would you express it? It's just about giving it a name in the code.